.github/workflows/LinuxRelease.yml

```diff
@@ -12,11 +12,8 @@ on:
   pull_request:
     paths-ignore:
       - '**.md'
-      - 'tools/jdbc/**'
-      - 'tools/nodejs/**'
-      - 'tools/juliapkg/**'
-      - 'tools/pythonpkg/**'
-      - 'tools/rpkg/**'
+      - 'tools/**'
+      - '!tools/pythonpkg/**'
```

Comment:
why do we not ignore python here?

Reply:
You are right, unneded. Thanks for the catch